scanner/node.go

```diff
@@ -1,18 +1,90 @@
 package scanner
 
+import (
+	"encoding/json"
+	"os"
+	"os/exec"
+	"strings"
+)
+
 func configureNode(sourceDir string) (*SourceInfo, error) {
 	if !checksPass(sourceDir, fileExists("package.json")) {
 		return nil, nil
 	}
 
+	data, err := os.ReadFile("package.json")
+	if err != nil {
+		return nil, nil
+	}
+
+	var result map[string]interface{}
+	err = json.Unmarshal(data, &result)
+	if err != nil {
+		return nil, nil
+	}
+
 	s := &SourceInfo{
-		Builder: "heroku/buildpacks:20",
-		Family:  "NodeJS",
-		Port:    8080,
+		Family: "NodeJS",
+		Port:   8080,
```

Comment:
Might be good to include information about using `PORT` and `8080` in the final output in case folks didn't follow standards here.

Reply:
Good idea.